src/components/DeckBuilderDryRunView/index.js

```diff
@@ -363,7 +359,7 @@ class DeckBuilderDryRunView extends React.Component {
                       .
                     </p>
 
-                    {this.props.hand.includes('N38') && (
+                    {this.props.deck.map(card => card.id).includes('N38') && (
```

Comment:
I think the hint should appear whenever Harvesters are in the deck, since it can be useful once the user has played the card without getting an option to add a card from their opponent's deck to theirs.